netbout-client/src/main/java/com/netbout/client/retry/ReAlias.java

```diff
@@ -70,7 +70,7 @@ public ReAlias(final Alias orgn) {
     @RetryOnFailure(
         verbose = false, attempts = Tv.TWENTY,
         delay = Tv.FIVE, unit = TimeUnit.SECONDS
-    )
+        )
```

Comment:
@mkordas Yes. It was looking better, and sure it is better, but again, violation of qulice from 0.15.2 and also with 0.16.2.